src/test/fuzz/policy_estimator.cpp

```diff
@@ -41,7 +41,14 @@ FUZZ_TARGET(policy_estimator, .init = initialize_policy_estimator)
                     return;
                 }
                 const CTransaction tx{*mtx};
-                block_policy_estimator.processTransaction(ConsumeTxMemPoolEntry(fuzzed_data_provider, tx), fuzzed_data_provider.ConsumeBool());
+                const CTxMemPoolEntry& entry = ConsumeTxMemPoolEntry(fuzzed_data_provider, tx);
+                const auto tx_info = NewMempoolTransactionInfo(entry.GetSharedTx(), entry.GetFee(),
+                                                               entry.GetTxSize(), entry.GetHeight(),
+                                                               /* m_from_disconnected_block */ fuzzed_data_provider.ConsumeBool(),
+                                                               /* m_submitted_in_package */ fuzzed_data_provider.ConsumeBool(),
+                                                               /* m_chainstate_is_current */ fuzzed_data_provider.ConsumeBool(),
+                                                               /* m_has_no_mempool_parents */ fuzzed_data_provider.ConsumeBool());
```

Comment:
you're changing how the fuzz inputs are processed here - maybe have `m_has_no_mempool_parents = ConsumeBool()` and make the others = false/false/true?

Reply:
Thanks, its now as you suggested